src/dashboards/Compact/index.tsx

```diff
@@ -516,7 +550,29 @@ const EnturDashboard = (): JSX.Element | null => {
                                         variant="light"
                                     />
                                 ) : null}
-                                <BikeTile stations={bikeRentalStations} />
+                                <MobilityTile
+                                    mobilityType={FormFactor.BICYCLE}
+                                />
+                            </div>
+                        ) : (
+                            []
+                        )}
+                        {carRentalStations && carRentalStations.length > 0 ? (
+                            <div
+                                key="rental-car"
+                                data-grid={getDataGrid(
+                                    numberOfStopPlaces + weatherCol,
+                                    maxWidthCols,
```

Comment:
tenker weatherCol ikke passer inn her men no clue hva getdatagrid gjør eller hvordan det fungerer soooo 🤷

Reply:
jeg bare kopierte det fra en annen komponent, ser ut som de andre bruker den så tenkte vi kunne ta den med her også just in case